config/test.json.template

```diff
@@ -97,5 +97,9 @@
       "DEFAULT": "7d",
       "AUTH": "7d"
     }
+  },
+  "limit": {
```

Comment:
Why are ratelimiting options needed while testing? Now that they aren't required in non prod environments these shouldn't matter.